src/pages/roadmap/index.module.scss

```diff
@@ -0,0 +1,163 @@
+@import 'src/styles/variables.module';
+@import 'src/components/Header/index.module';
+
+.content {
+  position: relative;
+  z-index: 1;
```

Comment:
Setting a z-index on `content` is somewhat counterintuitive.

I thought about why this was done - it's likely because the assets provided by the designers were not as expected (embellisheds not svg), so this implementation was forced.

You can try using https://image.online-convert.com/convert-to-svg to obtain a proper resource.